mix.exs

```diff
@@ -52,8 +52,8 @@ defmodule Bless.MixProject do
       # docs
       {:ex_doc, ">= 0.0.0", only: :dev, runtime: false, optional: true},
       # test
-      {:chaps, ">= 0.15.0", only: :test, optional: true},
-      {:credo, ">= 0.0.0", only: :test, runtime: false, optional: true}
+      {:chaps, ">= 1.0.0", only: :test, optional: true},
+      {:credo, ">= 1.7.0", only: :test, runtime: false, optional: true}
```

Comment:
These are the most recent. I'm not clear if this could be really painful in consumer applications... If so, may need to lower these to more reasonable versions? I didn't notice any problems in agora, which I tested against this.